WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -177,25 +177,35 @@ private extension PersonViewController {
         alert.addCancelActionWithTitle(cancelTitle)
 
         alert.addDestructiveActionWithTitle(removeTitle) { action in
-            self.removePerson()
+            self.deleteUser()
         }
 
         alert.presentFromRootViewController()
     }
 
-    func removePerson() {
+    func deleteUser() {
+        guard let user = user else {
+            DDLogSwift.logError("Error: Only Users can be deleted")
```

Comment:
We can keep the log for release builds, but for debug builds this should be an assertion failure since it clearly means something's wrong with the code logic.